Formula/a/asdf.rb

```diff
@@ -16,38 +16,17 @@ class Asdf < Formula
     sha256 cellar: :any_skip_relocation, all: "a94dd362ce5c3a818f4fa56607eac3c616a3c1191e9f41480093bda2dc308af4"
   end
 
-  depends_on "autoconf"
-  depends_on "automake"
-  depends_on "coreutils"
-  depends_on "libtool"
-  depends_on "libyaml"
+  depends_on "go" => :build
+
   depends_on "openssl@3"
-  depends_on "readline"
-  depends_on "unixodbc"
 
   def install
-    bash_completion.install "completions/asdf.bash" => "asdf"
-    fish_completion.install "completions/asdf.fish"
-    zsh_completion.install "completions/_asdf"
-    libexec.install Dir["*"]
-
-    bin.write_exec_script libexec/"bin/asdf"
-  end
-
-  def caveats
-    <<~EOS
-      To use asdf, add the following line (or equivalent) to your shell profile
-      e.g. ~/.profile or ~/.zshrc:
-        . #{opt_libexec}/asdf.sh
-      e.g. ~/.config/fish/config.fish
-        source #{opt_libexec}/asdf.fish
-      Restart your terminal for the settings to take effect.
-    EOS
+    system("go", "build", *std_go_args(ldflags: "-s -X main.version=#{version}"), "./cmd/asdf")
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}"), "./cmd/asdf"
```

Reply:
PR updated.

Reply:
Thanks. Looks like you missed the `-w` from my suggestion. We usually pass `-s -w` to reduce binary size.

Reply:
Sorry about that. Updated the PR again to add the `-w` flag.